### PR TITLE
status: unattached status output headers alignment off by one

### PR DESCRIPTION
**Pull request by blackboxsw**
Fixes: #806

### CODE REVIEW

File: uaclient/status.py [hunk 1]
```diff
@@ -198,8 +198,8 @@ def format_tabular(status: "Dict[str, Any]") -> str:
         content = [
             STATUS_UNATTACHED_TMPL.format(
                 name="\nSERVICE",
-                available="AVAILABLE",
-                description="DESCRIPTION",
+                available=" AVAILABLE",
```
Comment (OddBloke) [inline uaclient/status.py]: Is the issue here actually that we have `\n` at the start of name, which takes up one "character" of the line?

Comment (blackboxsw) [reply uaclient/status.py]: absolutely right. good suggestion. I've added a unit test and fix the change properly.

File: tox.ini [hunk 2]
```diff
@@ -19,7 +19,7 @@ commands =
     mypy: mypy --python-version 3.5 uaclient/
     mypy: mypy --python-version 3.6 uaclient/
     mypy: mypy --python-version 3.7 uaclient/
-    black: black --check --diff uaclient/
```
Comment (OddBloke) [inline tox.ini]: Why?

(This should at least be a separate commit capturing the rationale.)

Comment (blackboxsw) [reply tox.ini]: bogus commit. removed and forgot to push.

File: uaclient/tests/test_cli_attach.py [hunk 3]
```diff
@@ -17,8 +17,12 @@
...
 BASIC_MACHINE_TOKEN = {
     "machineTokenInfo": {
-        "contractInfo": {"name": "mycontract", "resourceEntitlements": []},
-        "accountInfo": {"name": "accountName"},
+        "contractInfo": {
+            "id": "contract-1",
+            "name": "mycontract",
+            "resourceEntitlements": [],
+        },
+        "accountInfo": {"id": "acct-1", "name": "accountName"},
```
Comment (OddBloke) [inline uaclient/tests/test_cli_attach.py]: It's not clear to me what's motivating this change being included in this PR. Am I missing some information?

Comment (blackboxsw) [reply uaclient/tests/test_cli_attach.py]: No motivation, I leaked it in from another branch accidentally. Removed and rechecked the full diff to confirm no stray commits from other branches.
